### ISSUE TITLE
How to calculate IPSS-R score?

### ISSUE THREAD

**Issue by SixPlusSeven** [papaemmelab/ipssm#7]
Hello, 

Could you tell me how to calculate IPSS-R score and IPSS-R score (age-adjusted) using ipssm package? I can see these two values on the web version, but I can't calculate them using the R package locally. In addition, I added two columns "ANC" and "AGE" to the IPSSMexample.csv file for local calculation.
`ID,HB,PLT,BM_BLAST,ANC,AGE,del5q,del7_7q,complex,CYTO_IPSSR,del17_17p,TP53mut,TP53maxvaf,TP53loh,MLL_PTD,FLT3,ASXL1,BCOR,BCORL1,CBL,CEBPA,DNMT3A,ETV6,EZH2,IDH1,IDH2,KRAS,NF1,NPM1,NRAS,RUNX1,SETBP1,SF3B1,SRSF2,STAG2,U2AF1,ETNK1,GATA2,GNB1,PHF6,PPM1D,PRPF8,PTPN11,WT1
pp347,9.6,281,9,3,19,0,0,0,Good,0,0,NA,0,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,0,NA,0,NA,0,0,NA,0,0`

Best,
Alex

**Comment by ElsaB** [comment]
Hello

You are correct, in the R package we did not implement the IPSS-R calculation, only the IPSS-M.
We added the IPSS-R in the web for the clinicians to have all in the information in one place.
For statisticians/analysts using R, IPSS-R is vey easy to calculate using the sum of the points attributed to the categories of features (as explained in https://www.mds-foundation.org/ipss-r-calculator/ ).

Best
Elsa
